app/controllers/events.server.controller.js

```diff
@@ -0,0 +1,107 @@
+const Event = require('mongoose').model('Event');
+
+function getErrorMessage(err) {
+  let message = '';
+
+  if (err.code) {
+    switch (err.code) {
+        case 11000:
+        case 11001: 
+          message = 'Email already exists';
```

Comment:
11000 and 11001 are duplicate key errors; this message should be something like 'Event already exists'